posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: "Deploy Java applications to IBM Cloud Code Engine using the Paketo Buildpack for Liberty"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars.githubusercontent.com/u/29545669?v=4
+author_github: https://github.com/kevin-ortega
+seo-title:  Deploy Java applications to IBM Cloud Code Engine (ICCE) using the Paketo Buildpack for Liberty - OpenLiberty.io
+seo-description: Learn to deploy your applications to ICCE easily with the Paketo Buildpack for Liberty
+blog_description: "Learn to deploy your applications to ICCE easily with the Paketo Buildpack for Liberty"
+open-graph-image: https://openliberty.io/img/blog/code_engine_build_step_details.png
```

Comment:
I would have suggested using [code_engine_build_strategy.png](https://github.com/OpenLiberty/blogs/pull/2746/files#diff-f654c4f7cbedd759fea54bf522d90fa0d1504ad6c068c5055c83e914101ec2f0)  instead as it seems more interesting